test/index.test.js

```diff
@@ -1148,7 +1168,8 @@ jobs:
 
             return scm.decorateUrl({
                 scmUri,
-                token: 'mytokenfortesting'
+                token: 'mytokenfortesting',
+                scmContext: 'github.com'
```

Comment:
Certainly, the function `decorateUrl` it self is not changed. However, the input of `decorateUrl` is validated with `decorateUrl` data-schema which is defined [here](https://github.com/tk3fftk/data-schema/blob/cd3ebbc3b047a0c7d35a300b163d6de91bf5e3d6/plugins/scm.js#L72). In the https://github.com/screwdriver-cd/data-schema/pull/155 , `scmContext` is added to `decorateUrl` data-schema. So I added additional param `scmContext`.